### DISCUSSION TITLE
I hope to get a unified UI useing to any platforms with no different display.

### DISCUSSION THREAD

**Discussion by MackYang** [dotnet/maui#3886] [Q&A]
Is it possible to develop a unified UI like Flutter to replace the native UI and completely solve the problem of UI differences between platforms?

**Comment by dhindrik** [comment]
Xamarin.Forms have the concept of Visuals and it seems like you will have that in .NET MAUI also, at least there are Material renders for iOS in the repo.

With that concept, it will be possible to replace the native UI.

**Comment by chris-ux** [comment]
Aside from Visual as mentioned by @dhindrik,  Doesn't Maui.Graphics.Controls fill this need?
https://github.com/dotnet/Microsoft.Maui.Graphics.Controls

**Comment by davidbuckleyni** [comment]
They are concentrating on c# for markup for this feature I  have asked several times for this and this is the answer they give which I guess is valid, but its closed to just the three platforms not balzor obv
